src/tests/ftest/util/ec_utils.py

```diff
@@ -61,6 +61,9 @@ def check_aggregation_status(pool, quick_check=True, attempt=20):
                 # Return immediately once aggregation starts for quick check
                 if quick_check:
                     return agg_status
+        else:
```

Comment:
(pylint-useless-else-on-loop) Else clause on loop without a break statement, remove the else and de-indent all the code inside it